sources/packages/backend/package.json

```diff
@@ -21,7 +21,7 @@
     "docker:start:queue-consumers": "node dist/apps/queue-consumers/main.js",
     "start:prod:load-test-gateway": "node dist/apps/load-test-gateway/main.js",
     "docker:start:load-test-gateway": "node dist/apps/load-test-gateway/main.js",
-    "test": "cross-env ENVIRONMENT=test jest --forceExit",
+    "test": "cross-env ENVIRONMENT=test jest --verbose --forceExit",
```

Comment:
what is the purpose `verbose`

Reply:
Without having verbose, we weren't getting this detail log at individual test level. We were just getting a log at test suite level.

Our E2E tests have it, but was missing in unit test.

![image](https://github.com/bcgov/SIMS/assets/54600590/5af60a28-26fc-400f-a4ba-37d47f5959f4)

Reply:
To output the test descriptions @ann-aot, same that we use for the E2Es 😉